client/components/tinymce/plugins/simple-payments/dialog/index.jsx

```diff
@@ -414,7 +446,7 @@ class SimplePaymentsDialog extends Component {
 		return (
 			<Dialog
 				isVisible={ showDialog }
-				onClose={ onClose }
+				onClose={ this.handleDialogClose }
```

Comment:
There are now two methods that close something: `handleDialogClose` and `handleFormClose`. I had to invent some good names 😄